_rules/focusable-no-keyboard-trap-standard-nav-a1b64e.md

```diff
@@ -80,13 +82,14 @@ This element is made [focusable][] by the `tabindex` attribute, even if it is no
 
 #### Failed Example 1
 
-This [focusable][] element creates a keyboard trap bringing focus to the `button`.
+This [focusable][] element creates a keyboard trap bringing focus to the `button`. Note that if one of the links is removed, the focus may jump to the browser UI before the timeout expires, at which point the `this.focus()` trap cannot trigger anymore.
 
 ```html
 <a href="#">Link 1</a>
 <button onblur="setTimeout(() => this.focus(), 10)">
 	Button1
 </button>
+<a href="#">Link 1</a>
```

Comment:
Irrelevant for understanding, but...

According to [focusable](https://www.w3.org/WAI/standards-guidelines/act/rules/a1b64e/proposed/#focusable) definition

> Exception: Elements that lose focus during a period of up to 1 second after gaining focus, without the user interacting with the page the element is on, are not considered focusable.

when focus is on the button element, first link (navigating backward) and second link (navigating forward) are not considered focusable. So in this scenario, when focus is on the "button" element, we've just one "focusable" element within the page that is the button itself. 

Should we replace "[focusable][]" with "part of [sequential focus navigation](https://html.spec.whatwg.org/multipage/interaction.html#sequential-focus-navigation)" in Applicability?

Reply:
@Jym77, per your answer above, I'm opening a new issue.

Reply:
I don't think that's needed 🤔 
I agree it is a bit weird that the focusability of the links depends on the state of the page (they are "focusable" only if the button isn't currently focused…) That's more a problem with the definition, I'd say.

But in any cases, the button still has a keyboard trap sa defined by this rule (focus can't move away).